extensions/binding/org.eclipse.smarthome.binding.weatherunderground/.classpath

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<classpath>
+	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.7"/>
```

Comment:
Shall I change `Bundle-RequiredExecutionEnvironment: JavaSE-1.7` to Bundle`-RequiredExecutionEnvironment: JavaSE-1.8` in MANIFEST.MF ?